nextflow.config

```diff
@@ -192,6 +200,10 @@ profiles {
 // Load nf-core custom profiles from different Institutions
 includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
 
+// Load nf-core/magmap custom profiles from different institutions.
+// TODO nf-core: Optionally, you can add a pipeline-specific nf-core config at https://github.com/nf-core/configs
```

Comment:
Remaining `TODO`. Won't lint. I think this line (and the next) can be deleted as we don't have any pipeline-specific institute profile (yet).